src/main/java/com/bp3/camunda/camunda7/C7RestConnector.java

```diff
@@ -60,9 +75,39 @@ public C7RestConnector(final HttpConnector httpConnector) {
                 .registerModule(new JavaTimeModule());
     }
 
+    /**
+     * Called in a PostConstruct because the properties will have been loaded at this point once the class
+     * has been constructed.
+     *
+     * The purpose of creating the External Task Client using the builder and not the @ExternalTaskSubscription
+     * annotation is, so we can configure the required topic this client will be subscribing to, which cannot be
+     * done using the @ExternalTaskSubscription annotation.
+     */
+    @PostConstruct
+    public void initTaskClient() {
+        log.debug("Initialising the External Task Client with the following properties:");
+        log.debug("Engine end point = {}".indent(LOG_INDENT), engineEndpoint);
```

Comment:
`indent()` is useful for multiline strings, but I think for what you have here it would be simpler and easier to read if you would just indent the log messags

```suggestion
        log.debug("    Engine end point = {}", engineEndpoint);
```

Reply:
I did have spaces in the log originally but saw this. But OK I can revert back.

Reply:
Reverted back to spaces in string.